examples/pagerduty-business-service/main.tf

```diff
@@ -1,15 +1,23 @@
+resource "random_id" "random_suffix" {
+  byte_length = 4
+}
+
+locals {
+  random_suffix = random_id.random_suffix.b64_url
+}
+
 module "mock_team" {
   source = "../../modules/pagerduty-team"
 
-  name        = "team - ${var.name}"
+  name        = "team - ${var.name} - ${local.random_suffix}"
```

Comment:
spacing in naming is inconsistent - for example in examples/honest-two-level-schedule/main.tf the name is:

```
  name        = "Example-${var.name}-${local.random_suffix}"
```

(no spacing around hyphens).

Not the end of the world, just a stylistic nit.

Reply:
fixed in latest commit please re-review 🙏